src/main/resources/default/assets/tycho/lib.js.pasta

```diff
@@ -2,13 +2,10 @@
 // A list of all licenses can be obtained via /licenses
 //
 
-___include("/assets/tycho/libs/jquery/jquery-3.5.1.min.js")
-___include("/assets/tycho/libs/popper/popper-1.14.3.min.js")
+___include("/assets/tycho/libs/popper/popper.min.js")
 ___include("/assets/tycho/libs/bootstrap/js/bootstrap.min.js")
```

Comment:
We could also directly include such files in the HTML page. That would enable the browsers to load source maps, thus facilitating debugging. Merging everything into one file is primarily for SEO, as far as I understand it, which is not so relevant on backend pages.